velox/dwio/common/CMakeLists.txt

```diff
@@ -72,4 +72,5 @@ target_link_libraries(
   velox_exec
   Boost::regex
   Folly::folly
-  glog::glog)
+  glog::glog
+  ${Protobuf_LIBRARIES})
```

Comment:
hey @majetideepak 

in case you curious the context for this oneline change,  it apply @yingsu00 's this commit https://github.com/facebookincubator/velox/pull/6965/commits/c3a9dcfc755c8e1c4e7bc3248505961b81e0058e to fix oene build issue https://github.com/facebookincubator/velox/issues/6641 when using ubuntu 2004 which is now i used for local development,

Reply:
Since this change is unrelated to this PR, let's move it to a separate PR.

Reply:
revert this part. thanks @majetideepak